base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -192,6 +196,7 @@ where D: Digest + Send + Sync
     // range_proof_mmr and kernel_mmr. CreateMmrCheckpoint and RewindMmr txns will be performed after the the storage
     // txns have been successfully applied.
     fn apply_mmr_txs(&self, tx: &DbTransaction) -> Result<(), ChainStorageError> {
+        debug!(target: LOG_TARGET, "DB apply mmr instruction received: {:?}", tx);
```

Comment:
You know that tx could be huge, right?

Reply:
These log messages could definitely be handy, but logging `tx` will cause way too much noise.